Apps/Sandcastle/gallery/Globe Materials.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Shader composition demos">
+    <meta name="cesium-sandcastle-labels" content="Tutorials, Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar">
+<div id="terrainMenu"></div>
+<div id="materialMenu"></div>
+<div id="zoomButtons"></div>
+<div id="sampleButtons"></div>
+<table><tbody>
+    <tr>
+        <td>Contour Spacing</td>
+        <td>
+            <input type="range" min="1.0" max="500.0 step="1.0" data-bind="value: spacing, valueUpdate: 'input'">
+        </td>
+    </tr>
+</tbody>
+</table>
+</div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+var cesiumTerrainProviderMeshes = new Cesium.CesiumTerrainProvider({
+    url : 'https://assets.agi.com/stk-terrain/v1/tilesets/world/tiles',
+    requestWaterMask : true,
+    requestVertexNormals : true
+});
+viewer.terrainProvider = cesiumTerrainProviderMeshes;
+
+viewer.scene.globe.enableLighting = true;
```

Comment:
The color ramps are better with lighting so you can see the terrain features.  The terrain all blends together without the shadows.

Reply:
The lighting does look fine most of the time, it's mainly when its completely dark that it's hard to see what's going on.

Reply:
I disagree.  For example, there are some random Appalachian mountains with and without lighting:

![image](https://user-images.githubusercontent.com/3451886/32578319-c6b368d8-c4aa-11e7-949c-06900bc3cc15.png)

![image](https://user-images.githubusercontent.com/3451886/32578325-cbb60cd2-c4aa-11e7-82a6-246f6e390959.png)

Having the lighting enabled makes a big difference.
To make sure the lighting always looks good with the preset views, we can add a line to set the clock time:
`viewer.clockViewModel.currentTime = Cesium.JulianDate.fromDate(new Date(2017, 8, 22, 0, 0, 0));`

Reply:
Ok yeah, setting the clock is good. I should have been clearer, my main issue was that whenever I would open the demo it would be night at Everest and just hard to see anything.

Reply:
I've pushed a change to set the clock time.